CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 ### Bug Fixes
 
+* [#2413](https://github.com/bbatsov/rubocop/issues/2413): Allow `%Q` for dynamic strings with double quotes inside them. ([@jonas054][])
```

Comment:
This should be in the `master` section. I cut `0.35.1` yesterday because of the caching bug.